x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/allowed_values/helpers.test.tsx

```diff
@@ -36,25 +31,16 @@ describe('helpers', () => {
     test('it returns false for a field that does NOT exist in `ecsMetadata`', () => {
       expect(
         hasAllowedValues({
-          ecsMetadata,
+          ecsMetadata: EcsFlatTyped,
           fieldName: 'does.NOT.exist',
         })
       ).toBe(false);
     });
-
-    test('it returns false when `ecsMetadata` is null', () => {
-      expect(
-        hasAllowedValues({
-          ecsMetadata: null, // <--
-          fieldName: 'event.category',
-        })
-      ).toBe(false);
-    });
```

Comment:
For reviewers:

This test was removed because it tests an impossible state. Details:
https://github.com/elastic/kibana/pull/185025/files#r1631497248